src/core/util/http.cc

```diff
@@ -44,63 +42,70 @@
 #include <vector>
 
 #include "router_context.h"
+#include "client/client_context.h"
+#include "client/address_book.h"
 #include "util/filesystem.h"
 
 namespace i2p {
 namespace util {
-namespace http {
+namespace http {  // TODO(anonimal): consider removing this namespace (its not needed)
```

Comment:
Its not needed. :smile:  We're nowhere near overcrowding the ```util``` namespace, and namespace ```http``` is too small to warrant its own namespace. I've been wanting to open a ticket re: namespace design for a long time now.  When that happens, I'm sure this will be mentioned.